docs/core/whats-new/dotnet-core-2-1.md

```diff
@@ -1,4 +1,4 @@
----
+﻿---
 title: What's new in .NET Core 2.1
 description: Learn about the new features found in .NET Core 2.1.
 author: rpetrusha
```

Comment:
I'll let @rpetrusha review this but we'd need to add the following metadata to this section
```yml
dev_langs: 
  - "csharp"
  - "vb"
```